pkg/qbert/qbert.go

```
@@ -149,6 +150,11 @@ func (c QbertImpl) AttachNode(clusterID, projectID, token string, nodeIDs []stri
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
+		respString, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			zap.S().Info("Error occured while converting responce body to string")
+		}
+		zap.S().Info(string(respString))
 		return fmt.Errorf("Unable to attach node to cluster, code: %d", resp.StatusCode)
```

Should we return the response error string as well?

---

Okay will return it instead of printing.